types/utils/setupWriteToDisk.d.ts

```diff
@@ -11,20 +11,14 @@ export = setupWriteToDisk;
  * @param {import("../index.js").Context<Request, Response>} context
  */
 declare function setupWriteToDisk<
-  Request_1 extends import("http").IncomingMessage,
-  Response_1 extends import("../index.js").ServerResponse
->(context: import("../index.js").Context<Request_1, Response_1>): void;
+	Request_1 extends import('http').IncomingMessage,
+	Response_1 extends import('../index.js').ServerResponse
+>(context: import('../index.js').Context<Request_1, Response_1>): void;
 declare namespace setupWriteToDisk {
-  export {
-    Compiler,
-    MultiCompiler,
-    Compilation,
-    IncomingMessage,
-    ServerResponse,
-  };
+	export { Compiler, MultiCompiler, Compilation, IncomingMessage, ServerResponse };
 }
-type Compiler = import("webpack").Compiler;
-type MultiCompiler = import("webpack").MultiCompiler;
-type Compilation = import("webpack").Compilation;
-type IncomingMessage = import("../index.js").IncomingMessage;
-type ServerResponse = import("../index.js").ServerResponse;
+type Compiler = import('webpack').Compiler;
+type MultiCompiler = import('webpack').MultiCompiler;
+type Compilation = import('webpack').Compilation;
+type IncomingMessage = import('../index.js').IncomingMessage;
+type ServerResponse = import('../index.js').ServerResponse;
```

Comment:
Looks like we need to run prettier on all types files, can you do it?

Reply:
Yes of course